string target_waypoint
int64 epoch
---
string[] source_waypoints
float64[] travel_times